tests/unit/lib/charms/vault_k8s/v0/test_vault_autounseal.py

```diff
@@ -459,10 +421,5 @@ def test_given_no_details_when_get_details_then_none_is_returned(self):
             relations=[vault_autounseal_relation],
             leader=True,
         )
-        action = scenario.Action(
-            "get-details",
-        )
-
-        action_output = self.ctx.run_action(action, state_in)
-
-        assert action_output.success is False
+        with pytest.raises(scenario.ActionFailed):
```

Comment:
Can you avoid to pin ops by using the ActionFailed from ops.testing instead?

```
from ops.testing import ActionFailed
```

This is what we did here for instance:
- https://github.com/canonical/sdcore-gnbsim-k8s-operator/blob/main/tests/unit/test_charm_simulation_action.py

Reply:
Done